Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
Having the destination interface `Countable`/`IteratorAggregate` seems strange to me. It makes it seem like there are multiple desinations when in fact, there are multiple Backups.